maraboupy/MarabouNetwork.py

```diff
@@ -116,6 +118,13 @@ def addRelu(self, v1, v2):
         """
         self.reluList += [(v1, v2)]
 
+    def addBilinear(self, v1, v2, v3):
+        """Function to add new equation to the network
```

Comment:
a new bilinear constraint to the network?

Reply:
fixed